### ISSUE TITLE
Allow users to define Kafka customer group id

### ISSUE THREAD

**Issue by mwysokin** [Graylog2/graylog2-server#3976]
## Expected Behavior

Be able to define a customer group id for Kafka inputs.

## Current Behavior

There is a hardcoded customer group id set to **graylog2**.
```
public class KafkaTransport extends ThrottleableTransport {
    public static final String GROUP_ID = "graylog2";
    ...
```

## Possible Solution

Allow user to supply the name through UI for each Kafka input and get rid of the default one whatsoever.

## Steps to Reproduce (for bugs)

1. Create any Kafka input.
2. Kafka consumer will connect with consumer group id **graylog2**.

## Context

If you have more than one Graylog setup inside a company and someone else connects to the same Kafka topic it might be troublesome. Also some ops teams may not allow generic customer group ids like graylog2.

## Your Environment

* Graylog Version:
2.2.3

**Comment by lpiet** [comment]
I can confirm running into the problem in real life. Another team subscribed to our kafka and "stole" some of our logging before we caught on to the problem and switched the inputs off.

 In addition it would make migrating a team of a shared graylog to a dedicated one a lot easier if two graylog instances could both consume from the same kafka in parrallel for a while to build up some logging history.

**Comment by jhaohai** [comment]
+1
We have two graylog clusters and need to subscribe logs from the same kafka cluster.

**Comment by bernd** [comment]
This has been implemented in #5867 and will be part of Graylog 3.1.
